### PR TITLE
Adapted to integration-test-commons

### PR DESCRIPTION
**Pull request by maltesander**
## Description

- Added CHANGELOG.md

## Review Checklist
- [x] Code contains useful comments

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,18 +4,12 @@
...
 Integration tests for the Stackable Operator for Apache Spark.
 
-This repository bundles integration tests for the [Stackable Operator](https://github.com/stackabletech/spark-operator) for Apache Spark. The integration tests should be deployed on the Stackable [test-dev-cluster](https://github.com/stackabletech/test-dev-cluster).
+This repository bundles integration tests for the [Stackable Operator](https://github.com/stackabletech/spark-operator) for Apache Spark. 
 
 ## Requirements
 
-- [test-dev-cluster](https://github.com/stackabletech/test-dev-cluster) to deploy the Stackable Agent, the Stackable Operator for Apache Spark and run the integrations tests. Please check the readme of test-dev-cluster on how to set it up correctly.
-- The tests use at least 3 and a maximum of 5 nodes. Therefore, the test-dev cluster should be initialized with 5 nodes / agents: `./init.sh debian spark-operator --scale agent=5`
+- The tests use at least 3 and a maximum of 5 nodes.
```
Comment (razvan) [inline README.md:11]: This was misleading to me. I started a cluster with 3 nodes and some tests failed (those requiring 5 nodes). Just say that they need 5 nodes.

Comment (maltesander) [reply README.md:11]: Yeah that is still a relict from the node label stuff. I tested with 3 and all tests are working. So i would just write: ...requires at least 3 nodes?

**Review by backstreetkiwi — APPROVED** [review]
Tests are green on CI: https://ci.stackable.tech/job/Spark%20Operator%20Integration%20Tests/20/

**Review by razvan — APPROVED** [review]
ok

**Pull request merged**